l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -e
+
+# ----------------------------------------------------------------------------
+#
+# Package       : libseccomp-golang
+# Version       : PACKAGE_VERSION=${1:-main}
+# Source repo   : https://github.com/seccomp/libseccomp-golang.git
+# Tested on     : UBI 8.5
+# Language      : GO
+# Travis-Check  : true
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Chandranana Naik <Naik.Chandranana@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=${2:-libseccomp-golang}
```

Comment:
this one should actually be a constant else the script will fail

Reply:
@seth-priya I was referring to gofuzz_ubi_8.5.sh script here it is defined likewise.

Reply:
Done